flink-table/flink-table-planner/src/main/java/org/apache/calcite/rex/RexFieldAccess.java

```diff
@@ -49,6 +49,12 @@
  *     AND gender = 'F')</pre>
  *
  * </blockquote>
+ *
+ * <p>FLINK modifications are at lines
+ *
+ * <ol>
+ *   <li>Should be removed after fixing CALCITE-6342 (Calcite 1.36.0): Lines 84-89
```

Comment:
Once Flink has upgraded to Calcite 1.36.0, do we need to remove linke 84-89?  Or can we remove the entire file?

Reply:
usually in such headers we put what is the difference between current version of class and the original Calcite one.
Since currently there is only one diff which should disappear together with upgrade to 1.36 then after upgrade this file will be identical to the one from Calcite => could be removed

unless for some reason more discrepancies appear...